### ISSUE TITLE
Getting sales history of items (forming base DB) doesn't work

### ISSUE THREAD

**Issue by Cfomodz** [Luois45/dmarket_bot#9]
![image](https://github.com/Luois45/dmarket_bot/assets/31261577/6d843ccd-f810-4fad-9583-f12a5646f553)

```
2024-01-12 17:20:07.435 | DEBUG    | modules.skinbase:filter_skins:58 - Game: Games.RUST. Get history for Adobe Door
2024-01-12 17:20:07.436 | DEBUG    | api.dmarketapi:generate_headers:32 - string_to_sign: GET/trade-aggregator/v1/last-sales?gameId=rust&title=Adobe+Door
2024-01-12 17:20:07.581 | ERROR    | modules.skinbase:filter_skins:76 - Exception in skinbase1 validation error for LastSales
LastSales
  Field required [type=missing, input_value={'sales': [{'price': '0.3...'orderAttributes': {}}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
2024-01-12 17:20:07.581 | ERROR    | modules.skinbase:filter_skins:76 - Exception in skinbase1 validation error for LastSales
LastSales
  Field required [type=missing, input_value={'sales': [{'price': '0.3...'orderAttributes': {}}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
2024-01-12 17:20:07.582 | DEBUG    | modules.skinbase:filter_skins:78 - Game: Games.RUST. Parsed 0 skins/items.
2024-01-12 17:20:07.582 | DEBUG    | modules.skinbase:filter_skins:58 - Game: Games.RUST. Get history for Adobe Garage Door
2024-01-12 17:20:07.583 | DEBUG    | api.dmarketapi:generate_headers:32 - string_to_sign: GET/trade-aggregator/v1/last-sales?gameId=rust&title=Adobe+Garage+Door
2024-01-12 17:20:07.714 | ERROR    | modules.skinbase:filter_skins:76 - Exception in skinbase1 validation error for LastSales
LastSales
  Field required [type=missing, input_value={'sales': [{'price': '0.4...'orderAttributes': {}}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
2024-01-12 17:20:07.714 | ERROR    | modules.skinbase:filter_skins:76 - Exception in skinbase1 validation error for LastSales
LastSales
  Field required [type=missing, input_value={'sales': [{'price': '0.4...'orderAttributes': {}}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
```

Recreate: run main.py

The issue is caused by the change in the response to the `/trade-aggregator/v1/last-sales` endpoint both in input expected and in expected response data.

This should be both straightforward and easy to debug and update to the current spec

**Comment by Cfomodz** [comment]
Oh, I forgot, I actually already fixed the change in expected input in #7 , so it's just handling the new response, likely with a modification of `offers.py`'s `add_to_sell()` function and/or the `CreateOffer` / `LastPrice` model(s)

**Comment by Cfomodz** [comment]
I don't even remember why at this point - this code is getting the best of me lol, but I fixed this issue in my current deployment (the one that was just merged in #7). It had something to do with the models, maybe naming changing in the response JSON.. Irdk, I know that was one bug but that feels like it was days ago even though I just did it yesterday. Sorry for the informal issues/comments 🙃 I'm used to just hacking things together by myself tbh..
